From 614e6dd19b4af0a59f683f216782d09512ea7426 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 14 Jan 2019 19:19:53 +0200
Subject: [PATCH] Revert "[SW-1886] pll: reduce Fvco to 1200MHz"

Since we w/a the stability issue in BTL, we no longer need to reduce
the Fvco to 1200MHz.
So revert the Fvco patch to use original values.

This reverts commit be8e536e068adaced0ab5aa8d5824c0d16ed3ba8.

Change-Id: Ie7a1b53e3564b586b575cc9b3199115329806c43
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/pll.c b/pll.c
index 1c2fa6c..1fc705b 100644
--- a/pll.c
+++ b/pll.c
@@ -59,7 +59,7 @@
 
 static struct hl_pll_cfg goya_pll_cfg[] = {
 	{.name="CPU", .base=(void*)mmCPU_PLL_BASE,
-	 .nr=0, .nf=23, .od=0, .nb=11, .div_factor={0,23,1,-1}},
+	 .nr=0, .nf=47, .od=1, .nb=23, .div_factor={0,23,1,-1}},
 	{.name="IC", .base=(void*)mmIC_PLL_BASE,
 	 .nr=0, .nf=63, .od=15, .nb=31, .div_factor={0,1,1,2}},
 	{.name="MME", .base=(void*)mmPSOC_MME_PLL_BASE,
